libs/ui/inputs/src/lib/navigation-button/navigation-button.component.html

```diff
@@ -1,6 +1,12 @@
 <button
-  class="group flex items-center justify-center p-1 bg-white opacity-80 box-shadow-xl shadow-[4.0px_8.0px_8.0px_rgba(0,0,0,0.38)] mix-blend-luminosity text-black rounded hover:opacity-100"
+  type="button"
+  class="relative backdrop-blur group flex items-center justify-center p-1 text-main rounded overflow-hidden shadow-2xl"
 >
-  <mat-icon class="material-symbols-outlined align-middle">{{ icon }}</mat-icon>
-  <span class="mx-2 hidden group-hover:inline">{{ label }}</span>
+  <div
+    class="absolute -z-0 inset-0 bg-white opacity-60 group-hover:opacity-100 transition duration-[20ms]"
```

Comment:
@fgravin Can do, but there's already a transition in the back to search button. Do you have something else in mind?

Reply:
The label extension, but forget it we will do that one day maybe :smile: